Model/Config/Source/PricingStrategy.php

```diff
@@ -13,8 +13,8 @@ class PricingStrategy implements \Magento\Framework\Option\ArrayInterface
     public function toOptionArray()
     {
         return [
-            ['value' => 'MINIMUM', 'label' => __('MINIMUM')],
             ['value' => 'MAXIMUM', 'label' => __('MAXIMUM')],
+            ['value' => 'MINIMUM', 'label' => __('MINIMUM')],
```

Comment:
Wanted to confirm my assumptions:
- this re-organized for aesthetic purposes
- The default (MAXIMUM) is set in the etc/config.xml file

thanks!

Reply:
No, not for aesthetic purpose. When both key and value are string ... if you do not use the default one as the first item it would not show you correct one in the beginning without  extra manipulation.

Reply:
thank you! I'm happy I checked.